### ISSUE TITLE
Conflict beetween Celery and  multiprocessing modules

### ISSUE THREAD

**Issue by thomasfillon** [ANR-DIADEMS/timeside-diadems#2]
In irit_DECAP.py the use of _multiprocessing  process_ is not compatible with the use of _Celery task_.

> daemonic processes are not allowed to have children

See . https://github.com/ANR-DIADEMS/timeside-diadems/blob/fb5abe8e2f334ca9cf8f0ead47bd1e2b075d3b9d/timeside/plugins/diadems/irit_DECAP.py#L123

And error tracebak:

```
worker_1 | Traceback (most recent call last):
worker_1 |   File "/opt/miniconda/lib/python2.7/site-packages/celery/app/trace.py", line 240, in trace_task
worker_1 |     R = retval = fun(*args, **kwargs)
worker_1 |   File "/opt/miniconda/lib/python2.7/site-packages/celery/app/trace.py", line 438, in __protected_call__
worker_1 |     return self.run(*args, **kwargs)
worker_1 |   File "/srv/src/timeside/timeside/server/tasks.py", line 29, in experience_run
worker_1 |     item.run(experience)
worker_1 |   File "/srv/src/timeside/timeside/server/models.py", line 253, in run
worker_1 |     pipe.run()
worker_1 |   File "/srv/src/timeside/timeside/core/processor.py", line 500, in run
worker_1 |     item.post_process()
worker_1 |   File "/srv/src/timeside-diadems/timeside/plugins/diadems/irit_DECAP.py", line 123, in post_process
worker_1 |     p.start()
worker_1 |   File "/opt/miniconda/lib/python2.7/multiprocessing/process.py", line 124, in start
worker_1 |     'daemonic processes are not allowed to have children'
worker_1 | AssertionError: daemonic processes are not allowed to have children

```

**Comment by thomasfillon** [comment]
See https://github.com/celery/celery/issues/1709

and http://stackoverflow.com/questions/30624290/celery-daemonic-processes-are-not-allowed-to-have-children

**Comment by thomasfillon** [comment]
Temporary fixed by d6668e24bd37c16796df
